src/fireblocks-sdk.ts

```diff
@@ -198,13 +199,96 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest("/v1/network_connections");
     }
 
+    /**
+     * Creates a network connection
+     * @param localNetworkId NetworkId for the local
+     * @param remoteNetworkId NetworkId for the remote
+     * @param routingPolicy of the connection
+     */
+    public async createNetworkConnection(localNetworkId: string, remoteNetworkId: string, routingPolicy?: RoutingPolicy) {
+        const body = { localNetworkId, remoteNetworkId, routingPolicy };
+        return await this.apiClient.issuePostRequest(`/v1/network_connections`, body);
+    }
+    
     /**
      * Gets a single network connection by id
      */
     public async getNetworkConnectionById(connectionId: string): Promise<NetworkConnectionResponse> {
         return await this.apiClient.issueGetRequest(`/v1/network_connections/${connectionId}`);
     }
 
+    /**
+     * Deletes a network connection
+     * @param connectionId the connectionId between the peers
+     */
+    public async removeNetworkConnection(connectionId: string) {
+        return await this.apiClient.issueDeleteRequest(`/v1/network_connections/${connectionId}`);
+    }
+
+    /**
+     * Set routing policy for networkId 
+     * @param routingPolicy  the policy
```

Comment:
redundant space between **routingPolicy** and **the**

Reply:
👍